cassandra/cassandra.go

```diff
@@ -38,6 +39,13 @@ const (
 	pluginType = plugin.PublisherPluginType
 
 	serverAddrRuleKey          = "server"
+	portRuleKey                = "port"
+	timeoutRuleKey             = "timeout"
+	connectionTimeoutRuleKey   = "connectionTimeout"
+	initialHostLookupRuleKey   = "initialHostLookup"
+	ignorePeerAddrRuleKey      = "ignoreRuleKey"
+	createKeyspaceRuleKey      = "createKeyspace"
```

Comment:
it seems that we don't need this property. The keyspace is default to `snap`. Keyspace name suffices.